### ISSUE TITLE
Tribes v1.1 Placeholder

### ISSUE THREAD

**Issue by frankchen07** [gitcoinco/web#5626]
### User Story

Gitcoin Tribes is currently in its 2nd iteration. We've received initial feedback on what the feature set might be for Tribes to be a success.

### Why Is this Needed

Tribes offers a recurring subscription for returning hackathon sponsors to take even greater advantage of the open source developers that they've worked with. 

### Description

With our customer research, we're funneling towards the initial feature set of tribes. 

https://docs.google.com/spreadsheets/d/1Em3usJO_8Cv8PS_Ura3Wrp23_xog4t1qstGI6_HdOCk/edit#gid=0

In short:

1. find and manage your existing open source tribe
2. take actions to best utilize your open source tribe

### Current Behavior

no feature set

### Expected Behavior'

defined featured set to build

### Definition of Done
TBD

**Comment by frankchen07** [comment]
needs a ux discussion for sprint 26 between @vs77bb, @willsputra, @PixelantDesign, & @frankchen07
